admin/admin-base.php

```diff
@@ -172,13 +171,16 @@ public function languages_page() {}
 	 * @return void
 	 */
 	public function admin_enqueue_scripts() {
+		$suffix = defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ? '' : '.min';
+
+		wp_enqueue_script( 'pll_admin', plugins_url( "/js/build/admin{$suffix}.js", POLYLANG_ROOT_FILE ), array( 'jquery' ), POLYLANG_VERSION, true );
+		wp_localize_script( 'pll_admin', 'pll_admin', array( 'ajax_filter' => $this->get_ajax_filter_data() ) );
```

Comment:
`wp_add_inline_script()` is the way to go regarding such data transfer.
See what we've done in Polylang Pro [here](https://github.com/polylang/polylang-pro/blob/81bbe2aecf54cc1e531f9e3bb5f958a94647f571/modules/Editors/Screens/Abstract_Screen.php#L105-L121).

Reply:
Done in [bdedea8](https://github.com/polylang/polylang/pull/1612/commits/bdedea8143bcad10147378afb852967a320ebe00).
I didn't add a global filter because there is already a filter in `get_ajax_filter_data()`. I don't think a filter for the whole object would be useful.

Reply:
Agree, no need to filter the value here.